garage/src/pages/RigDetails/index.tsx

```diff
@@ -199,7 +199,8 @@ export const RigDetails = () => {
   const userOwnsRig = useMemo(() => {
     return (
       !!actingAsAddress &&
-      actingAsAddress.toLowerCase() === owner?.toLowerCase()
+      actingAsAddress.toLowerCase() ===
+        ((owner as any)?.toLowerCase() as string)
```

Comment:
This is one of the obvious issues—I was running into issues about `toLowercase` being used incorrectly. E.g., see the logs upon requesting Rig info: https://rigs-garage-dj48r6f04-tableland.vercel.app/rigs/356